sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -229,7 +229,12 @@ public enum UserRight {
 	ENVIRONMENT_PATHOGEN_TEST_CREATE(UserRightGroup.ENVIRONMENT, UserRight._ENVIRONMENT_SAMPLE_VIEW),
 	ENVIRONMENT_PATHOGEN_TEST_EDIT(UserRightGroup.ENVIRONMENT, UserRight._ENVIRONMENT_SAMPLE_EDIT),
 	ENVIRONMENT_PATHOGEN_TEST_DELETE(UserRightGroup.ENVIRONMENT, UserRight._ENVIRONMENT_SAMPLE_VIEW),
-	
+
+	SELF_REPORT_VIEW(UserRightGroup.SELF_REPORT),
+	SELF_REPORT_CREATE(UserRightGroup.SELF_REPORT, UserRight._SELF_REPORT_VIEW),
+	SELF_REPORT_EDIT(UserRightGroup.SELF_REPORT, UserRight._SELF_REPORT_VIEW),
+	SELF_REPORT_DELETE(UserRightGroup.SELF_REPORT, UserRight._SELF_REPORT_VIEW),
+
 	DOCUMENT_VIEW(UserRightGroup.DOCUMENT),
```

Comment:
There is an acceptance criteria in #13065, to display SELF_REPORT related user rights only when the SELF_REPORT feature configuration is enabled.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/edac40f4-5b80-4c47-aa05-076f4cf95d68)
Now the user rights are displayed without taking into consideration the feature configuration. 
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9111974e-2336-426b-82ca-eb9e88e6a4f4)
![Uploading image.png…]()

Reply:
Is not related to this class but there is another acceptance criteria in #13065 related to the display of the Self reporting menu item.
If this requirement will be implemented in #13068, the acceptance criteria should be moved there.
Otherwise should be implemented in this PR.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/698fe9fb-b83a-4fd1-b4cd-d5835a7a842d)

Reply:
Acceptance criteria changed:
No need to show/hide self report rights based on feature status, never did for any of the rights
Menu will be created by https://github.com/SORMAS-Foundation/SORMAS-Project/issues/13068, so the acceptance criteria related to it is moved to that ticket